pkg/types/vsphere/platform.go

```diff
@@ -174,6 +183,15 @@ type FailureDomain struct {
 	// Topology describes a given failure domain using vSphere constructs
 	// +kubebuilder:validation:Required
 	Topology Topology `json:"topology"`
+
+	// Type is the type of failure domain, the current values are "Datacenter", "ComputeCluster" and "HostGroup"
```

Comment:
So the documentation for `Zone` IMO needs to be updated. It currently says:

```
 zone defines a VSpherePlatformFailureDomain which
	// includes the name of the vCenter tag, the failure domain type
	// and the name of the vCenter tag category
```

Which is perhaps incorrect. We know that `tag-category` can only be `openshift-zone`. I do not think we have made this configurable. Also for `hostGroup`, this should be update to reflect something like:

```
Zone defines tag associated with `openshift-zone` category with which hosts in a given hostgroup has been tagged with
```

Reply:
@gnufied updated wdyt? 3145cca697f926c921205781cda3b0519d6e192e

Reply:
lgtm